apps/sleepphasealarm/interface.html

```diff
@@ -0,0 +1,108 @@
+<html>
+  <head>
+    <link rel="stylesheet" href="../../css/spectre.min.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/chart.js@2.8.0/dist/Chart.min.css">
+  </head>
+  <body>
+    <p>Please select a wakeup day:</p>
+    <div class="form-group">
+      <select id="day" disabled class="form-select">
+        <option selected disabled>No day</option>
+      </select>
+    </div>
+    <div class="chart-container">
+      <canvas id="sleepChart"></canvas>
+    </div>
+
+    <script src="../../core/lib/interface.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/chart.js@3.7.1/dist/chart.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/chartjs-adapter-date-fns@2.0.0/dist/chartjs-adapter-date-fns.bundle.min.js"></script>
```

Comment:
fetching files from CDN instead of downloading
them. Not very privacy friendly, but eg. openstmap 
also fetched from cdn.

Reply:
Thanks - I think that is probably fine. I guess we could maybe start hosting the files locally, but it feels like for `interface` files this is fine - it's not like they get loaded every time you run the app loader